SCIRun5ModuleGeneration.md

```diff
@@ -321,7 +313,7 @@ We will show the examples located in `src/Interface/Modules/Template` as example
 The module design file is an xml file that describes the UI structure. This file can be created and edited in the Qt editor. Figure 2.1 shows the example template *ModuleDesignerFile.ui* within the Qt editor.
 As shown, the user can interactively modify the placement of the widgets in the window. The Widget Box window allows the user to choose and place new objects within the window. The Property Editor allows for the modification of properties of the various objects and widgets within the UI, including size, type of input, names, etc. With the Object Inspector window, the hierarchy and organization of the UI can be changed.
 
-![Template module interface design file as seen in the Qt editor.](SCIRunImages/QT-editor.png)
+![Template module interface design file as seen in the Qt editor.](SCIRun5ModuleGeneration_figures/QT-editor.png)
```

Comment:
can you try to use html instead of markdown for this:
```
<figure id="editor">
  <img src="SCIRun5ModuleGeneration_figures/QT-editor.png" alt="Template module interface design file as seen in the Qt editor.">
  <figcaption>Figure 2.1 Template module interface design file as seen in the Qt editor.</figcaption>
</figure>
```